packages/fake-data/src/fakeExperiences.ts

```diff
@@ -21,8 +21,8 @@ import {
   GovContractorType,
   CSuiteRoleTitle,
 } from "@gc-digital-talent/graphql";
-import { fakeDepartments } from "@gc-digital-talent/fake-data";
 
+import fakeDepartments from "./fakeDepartments";
```

Comment:
Is this change for consistency, performance, or preference? Do other instances need to change as well?